### PR TITLE
Mike Gotta webpage

### PR DESCRIPTION
**Pull request by gottamike**
overhauled my entire personal webpage

### CODE REVIEW

**Comment by Sawahh** [comment]
Your footer isn't exactly behaving like a footer, maybe fix that?
image

File: students/mike.html [hunk 1]
```diff
@@ -4,10 +4,26 @@
...
+
     <title>Methods 2022 - Students - Mike Gotta</title>
 </head>
 <body>
     <h1>Mike's super special page</h1>
+
+    <p>Hello! My name is Mike Gotta, I am a junior studying computer science. I can't remember much about html or css so you are witnessing the very best front end code I can pull off. Some things I really enjoy include, camping, gold panning, exploring, shooting, and sleeping in. I've got a couple of cool links you can check out if you'd like!</p>
+     <ul>
+        
+        <li> <a href="https://github.com/gottamike">My Github</a></li>
```
Comment (Sawahh) [inline students/mike.html:17]: Add linkedin too? Unless you don't have it which is fine

**Review by Phred7 — APPROVED** [review]

**Comment by turnerburchard** [comment]
Close or merge this request?

**Comment by gottamike** [comment]
> Close or merge this request?
Let me see if I can fix this footer, it seems to be working on my end, but I dont want to risk pushing to main and have it not working.

**Pull request merged**